packages/runtime/container-runtime/src/test/id-compressor/numericUuid.spec.ts

```diff
@@ -30,7 +30,7 @@ describe("NumericUuid", () => {
 		assert.strictEqual(isStableId("8e8fec9a10ea4d158308ed35bc7f1e66"), false);
 		assert.strictEqual(isStableId("8e8fec9a-10ea-4d15-8308-ed35bc7f1e66"), true);
 		[...new Array(16).keys()]
-			.map((n) => [n, n.toString(16)])
+			.map<[number, string]>((n) => [n, n.toString(16)])
```

Comment:
The inferred type was (number | string)[], which was permissible in TS4 where number/string comparisons were allowed.